test/helpers/fixture_set.rb

```diff
@@ -97,6 +97,14 @@ def assert_configmap_present(cm_name, expected_data)
       assert_equal expected_data, configmaps.first["data"].to_h
     end
 
+    def assert_cronjob_exists(job_name, app_name)
```

Comment:
`app_name` is a property of these instances with an accessor; you don't need to pass it in.

Reply:
Thx for the tip.